src/main/java/org/motechproject/mots/domain/enums/UserPermission.java

```diff
@@ -0,0 +1,6 @@
+package org.motechproject.mots.domain.enums;
+
+public enum UserPermission {
```

Comment:
I'd keep it in different package - no need to differentiate on the type, rather on its purpose. Something like domain.auth or domain.security.